assets/ar/semantics/STS/STSTrack2_GPT35_ZeroShot.py

```diff
@@ -1,14 +1,14 @@
 import os
 import re
 
-from llmebench.datasets import STSArSemEval17Track2Dataset
+from llmebench.datasets import SemEval17T2STSDataset
 from llmebench.models import GPTModel, RandomGPTModel
 from llmebench.tasks import STSTrack2Task
 
 
 def config():
     return {
-        "dataset": STSArSemEval17Track2Dataset,
+        "dataset": SemEval17T2STSDataset,
         "dataset_args": {},
         "task": STSTrack2Task,
```

Comment:
I think we can merge `STSTrack1Task`and `STSTrack2Task` into an `STSTask`

Reply:
Yes, same evaluation approach for both.

Reply:
Done in e812c9d